packages/flutter_tools/test/commands.shard/hermetic/build_linux_test.dart

```diff
@@ -362,17 +362,17 @@ clang: error: linker command failed with exit code 1 (use -v to see invocation)
     expect(configLines, containsAll(<String>[
       'file(TO_CMAKE_PATH "$_kTestFlutterRoot" FLUTTER_ROOT)',
       'file(TO_CMAKE_PATH "${fileSystem.currentDirectory.path}" PROJECT_DIR)',
-      r'  "DART_DEFINES=\"foo.bar%3D2,fizz.far%3D3\""',
-      r'  "DART_OBFUSCATION=\"true\""',
-      r'  "EXTRA_FRONT_END_OPTIONS=\"--enable-experiment%3Dnon-nullable\""',
-      r'  "EXTRA_GEN_SNAPSHOT_OPTIONS=\"--enable-experiment%3Dnon-nullable\""',
-      r'  "SPLIT_DEBUG_INFO=\"foo/\""',
-      r'  "TRACK_WIDGET_CREATION=\"true\""',
-      r'  "TREE_SHAKE_ICONS=\"true\""',
-      '  "FLUTTER_ROOT=\\"$_kTestFlutterRoot\\""',
-      '  "PROJECT_DIR=\\"${fileSystem.currentDirectory.path}\\""',
-      r'  "FLUTTER_TARGET=\"lib/other.dart\""',
-      r'  "BUNDLE_SKSL_PATH=\"foo/bar.sksl.json\""',
+      r'  "DART_DEFINES=foo.bar%3D2,fizz.far%3D3"',
+      r'  "DART_OBFUSCATION=true"',
+      r'  "EXTRA_FRONT_END_OPTIONS=--enable-experiment%3Dnon-nullable"',
+      r'  "EXTRA_GEN_SNAPSHOT_OPTIONS=--enable-experiment%3Dnon-nullable"',
+      r'  "SPLIT_DEBUG_INFO=foo/"',
+      r'  "TRACK_WIDGET_CREATION=true"',
+      r'  "TREE_SHAKE_ICONS=true"',
+      '  "FLUTTER_ROOT=$_kTestFlutterRoot"',
+      '  "PROJECT_DIR=${fileSystem.currentDirectory.path}"',
+      r'  "FLUTTER_TARGET=lib/other.dart"',
+      r'  "BUNDLE_SKSL_PATH=foo/bar.sksl.json"',
```

Comment:
Nit: you should be able to drop all the `r` prefixes here; they were to simplify the quote escaping.